raiden-ts/src/polyfills.ts

```diff
@@ -25,3 +25,13 @@ request((opts: object, cb: ReqCb) => {
     },
   });
 });
+
+import wrtc from 'wrtc';
+if (!('RTCPeerConnection' in globalThis)) {
```

Comment:
`globalThis` was introduced after node10 IIRC, but we may need to support node10 because it's the last version with official armv6 releases which could be useful for IoT things. 

I remember running into this issue when working with the demo train

Reply:
@eorituz said that armv7 support is probably sufficient because armv6 is too old

Reply:
IIRC I was able to run node 12 on an armv6 but anyway. We can still figure out how to hack a version for armv6 if we really need to.